IDE/XCODE/README.md

```diff
@@ -54,6 +54,13 @@ device build. Both are debug builds.
 
 You can make an archive for a device, as well. That is a release build.
 
+## Known issues:
+
+When building for older iPhone models that support the i386 architecture some
```

Comment:
This note is wrong.  i386 does not assume 64bit registers.  The first step is not to disable asm.  See the old REAMDE (changelog?) for the steps.

Reply:
Resolved.